x11-libs/libfm/libfm-1.3.0.2.recipe

```diff
@@ -12,16 +12,24 @@ COPYRIGHT="
 	Andriy Grytsenko (LStranger)
 	"
 LICENSE="GNU GPL v2"
-REVISION="1"
+REVISION="2"
 SOURCE_URI="https://github.com/lxde/libfm/archive/$portVersion.tar.gz"
```

Comment:
We could switch this package to use the release source instead

    https://downloads.sourceforge.net/pcmanfm/libfm-$portVersion.tar.xz

It contains a pre-generated `configure` script, so we can get rid of autoconf and friends

Reply:
download from sf ends up in a double check: CDPATH="${ZSH_VERSION+.}:" && cd . && /bin/sh /sources/libfm-1.3.0.2/missing autoconf